buildpipeline/centos.6.groovy

```diff
@@ -34,7 +34,7 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:centos-6-376e1a3-201743110
         if (params.TestOuter) {
             additionalArgs = '-Outerloop'
         }
-        sh "LD_LIBRARY_PATH=/usr/local/lib ./build-tests.sh -buildArch=x64 -${params.CGroup} -SkipTests ${additionalArgs} -- /p:ArchiveTests=true /p:EnableDumpling=true /p:PortableBuild=false"
+        sh "LD_LIBRARY_PATH=/usr/local/lib ./build-tests.sh -buildArch=x64 -${params.CGroup} -SkipTests ${additionalArgs} -- /p:ArchiveTests=true /p:EnableDumpling=true /p:PortableBuild=false /p:IncludeVSTestReferences=false"
```

Comment:
I don't feel a 100% positive about having to do this, it is unfortunate that for CI builds (I guess for official builds) we would have to do this. I think we could probably find a better mechanism to decide whether we want to include this references or not. Maybe with like a `IsCI/OfficialBuild=true`? 

@weshaggard what are your thoughts?

Reply:
I agree we should avoid needing to explicitly pass this to all the CI legs. If it is interesting to disable for a particular scenario then we should try to detect and handle it in the targets so all scenarios (dev, ci, official) use the same logic.

Reply:
Also as far as I can tell the PR title doesn't seem to match the change. You appear to be disabling this for more then just aot.

Reply:
Do you suggest to pass in a property which tells that we are invoking from CI / official builds? I guess we don't have one already?

Reply:
Ok found the `IsCIBuild` property.

Reply:
> Is there any strong reason why you need to disable those references when in Ci/Official build?

Yes, several:
- Reduced disk space required for helix test submission upload
- No launchSettings.json generation in the src directory which could probably cause issues at some point.
- Less packages to restore and less work to be done which results in faster CIs.

> This IsCIBuild property is only set to true by the code coverage CI leg script.

Why not also by the others?

> whether you're in an official build you can just check if $(OfficialBuildId) != '' then you're in an official build.

official build means a daily build from mc.dot.net or also CI here?

Reply:
> Why not also by the others?

I guess because it wasn't needed. However this will be changing soon when @weshaggard merges his PR to move to arcade. 

> official build means a daily build from mc.dot.net or also CI here?

Yes, daily builds 😄

Reply:
The property you want to use that will be the one passed once we move to arcade is `ContinuousIntegrationBuild` -- so might be worth making the change in this PR and then whenever we get the change this would start avoiding those references in CI builds.

Reply:
Got it, thanks.